File: tmp_check_cols.js
import pg from 'pg';
import dotenv from 'dotenv';
dotenv.config();

const tableName = process.argv[2] || 'schools_IERN';

async function checkColumns() {
  const pool = new pg.Pool({ 
    connectionString: process.env.DATABASE_URL,
    ssl: { rejectUnauthorized: false }
  });
  try {
    const res = await pool.query("SELECT column_name FROM information_schema.columns WHERE table_name = $1", [tableName]);
    console.log(`COLUMNS for ${tableName}:`, res.rows.map(r => r.column_name).sort());
  } catch (err) {
    console.error("ERROR:", err.message);
  } finally {
    await pool.end();
  }
}

checkColumns();

File: tmp_check_cols.py
import os
import psycopg2
from dotenv import load_dotenv

load_dotenv()
database_url = os.getenv('DATABASE_URL')
conn = psycopg2.connect(database_url)
cur = conn.cursor()
cur.execute("SELECT column_name FROM information_schema.columns WHERE table_name = 'ph_schools'")
print("ph_schools columns:")
print([row[0] for row in cur.fetchall()])

cur.execute("SELECT column_name FROM information_schema.columns WHERE table_name = 'all_locations'")
print("\nall_locations columns:")
print([row[0] for row in cur.fetchall()])
cur.close()
conn.close()
